.github/workflows/rws-submissions-checks.yml

```diff
@@ -46,20 +46,25 @@ jobs:
           CONTENTS: ${{ steps.read_results.outputs.contents }}
         run: echo "$CONTENTS"
       - name: Create Comment
-        if: steps.read_results.outputs.contents != 'success'
+        if: steps.read_results.outputs.contents != 'success' && ${{ !startsWith(steps.read_results.outputs.contents, 'Warning')}}
         run: |
          echo "It appears you have failed some tests. Here are your results:" > message.txt
          cat results.txt >> message.txt
-      - name: Comment if sucess
+      - name: Comment if success
         if: steps.read_results.outputs.contents == 'success'
         run: echo "Looks like you've passed all of the checks!" >> message.txt
+      - name: Comment if warning
+        if: startsWith(steps.read_results.outputs.contents, 'Warning')
+        run: |
+          echo "You have passed all of the checks, but there are one or more warnings associated with your submission." > message.txt
+          cat results.txt >> message.txt
       - name: Write Comment on PR
         uses: mshick/add-pr-comment@v2
         with:   
           message-path: "message.txt" 
           refresh-message-position: true
       - name: Fail or Succeed
-        if: steps.read_results.outputs.contents != 'success'
+        if: startsWith(steps.read_results.outputs.contents, 'It appears you have failed')
```

Comment:
Can all of this be moved into the "Create Comment" step, so that we don't have to duplicate the "It appears you have failed" substring? Duplicating that makes this brittle, it'll break if someone changes that string and doesn't realize there's a dependency here.

Reply:
bari saya petunjuk. bantu saya dalam proses ini.     semua keputusan di tangan saya.  nama anda akan saya kait dalam keberhasilan saya.